sdk/tables/azure-data-tables/samples/README.md

```diff
@@ -0,0 +1,77 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-data-tables
```

Comment:
this should be azure-table-storage 
Source: https://review.docs.microsoft.com/en-us/new-hope/information-architecture/metadata/taxonomies?branch=master#product